language_tour.md

```diff
@@ -366,13 +366,37 @@ The patterns are not exhaustive. Missing cases:
        | }
 ```
 
+### Constants
```

Comment:
I think the explanation you provide here is great. Maybe you could add a "for instance" for the second one, i.e., "which, for instance, can be useful to set the size of a collection to the size of the biggest collection provided by different parties:"